lib/dataElementListener.js

```diff
@@ -192,8 +192,9 @@ class DataElementImporter extends SHRDataElementParserListener {
     if (ctx.count()) {
       [min, max] = this.getMinMax(ctx.count());
     }
+    let idText = '';
```

Comment:
Since you're not using `idText` in the CodeConstraint logic anymore, you can revert it back to local `const` statements (like it was before).